### PR TITLE
1-dot-waifu

### PR DESCRIPTION
**Pull request by vikiival**
## BIT OF KUSAMA WAIFU

- :zap: update metadata for claim
- :truck: rename claim
- :zap: miniqueries
- :construction: dot-drop
- :zap: no try again in component
- :wrench: giving $35 away
- :zap: conditional header layour
- :zap: conditional banner
- :bug: remove seconds
- :label: undefined is not assignable to string
- :broom: cleared components
- :wrench: correct constant
- :wrench: correct constant
- :construction: almost works
- :zap: 80% works

### CODE REVIEW

File: components/collection/drop/TokenImportButton.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<template>
+  <Loader v-if="isLoading" v-model="isLoading" />
+  <NeoButton
+    v-else
+    class="mb-2 mt-4 mint-button"
+    variant="secondary"
+    label="Teleport Token"
+    @click.native="handleTokenImport">
```
Comment (roiLeo) [inline components/collection/drop/TokenImportButton.vue:8]: check if account connected beforehand

**Review by roiLeo — APPROVED** [review]
- i18
- svg
- (collection `/stmn/collection/29` empty?)

otherwise lgtm

**Comment by yangwao** [comment]
image

got this, then suddenly bump some success notification.. and no redirekt?

**Comment by yangwao** [comment]
aaah this is supposed to be clickable button 

image

**Comment by yangwao** [comment]
Clicked and this is what I've got

image

**Comment by yangwao** [comment]
metadata seems legit
- https://image-beta.w.kodadot.xyz/ipfs/bafkreieewi4lphziaxvf7fi7g5bz5mkkrhra24o3d44jncyhrbxunosihm

**Comment by helloitsdamsky** [comment]
Current state: 
image

**Comment by vikiival** [comment]
Screenshot 2023-06-28 at 13 44 21

**Comment by vikiival** [comment]
Maybe some cache?

File: components/collection/unlockable/UnlockableContainer.vue [hunk 2]
```diff
@@ -298,12 +298,16 @@ const handleSubmitMint = async () => {
...
       toast('mint success')
-      justMinted.value = `${collectionId}-${res.result.sn}`
       scrollToTop()
+      return `${collectionId}-${res.result.sn}`
     })
+    // 20s timeout
+    setTimeout(() => {
+      isLoading.value = false
+      justMinted.value = id
+    }, 40000)
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableContainer.vue:308]: This is not 20 sec 😃 , what about your solution to fetch data using polkadot api?

File: components/collection/unlockable/UnlockableLoader.vue [hunk 3]
```diff
@@ -72,7 +72,7 @@ const displaySeconds = computed(() => {
 
 const twitterText = computed(
   () =>
-    'Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don\'t miss your chance! \n\n https://kodadot.xyz/stmn/drops/free-drop'
+    "Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don't miss your chance! \n\n https://kodadot.xyz/stmn/drops/free-drop"
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLoader.vue]:
```suggestion
    'Just minted an exclusive NFT with unlockable items on @Kodadot! 🎉 So excited to add this unique collectible to my collection. Don\'t miss your chance! \n\n https://kodadot.xyz/stmn/drops/free-drop'
```
again?

File: components/collection/unlockable/UnlockableContainer.vue (hunk 2 above)
Comment (vikiival) [reply components/collection/unlockable/UnlockableContainer.vue:308]: > what about your solution to fetch data using polkadot api?

This should have been done a lot of hours ago ^-^. Spend the night debugging fancy stuff. Will refactor after decoded

**Comment by yangwao** [comment]
And second one for 1 DOT ? 

image

**Comment by vikiival** [comment]
> And second one for 1 DOT ?

Will come soon. Let's merge if it works for you

**Comment by yangwao** [comment]
isn't this PR about that one? 
image

**Comment by yangwao** [comment]
Can I test that one 1 DOT ?

https://deploy-preview-6342--koda-canary.netlify.app/stt/drops/dot-drop

https://deploy-preview-6342--koda-canary.netlify.app/stt/drops/free-drop

**Comment by vikiival** [comment]
> https://deploy-preview-6342--koda-canary.netlify.app/stt/drops/free-drop

You have bad url

**Comment by vikiival** [comment]
> isn't this PR about that one? 

Yes + fixes free-drop

**Comment by yangwao** [comment]
> And second one for 1 DOT ?

whenever we will have both drops here https://deploy-preview-6342--koda-canary.netlify.app/stmn/drops we can merge it and release it to production. QR codes are being ready for distribution at Decoded

**Comment by yangwao** [comment]
image

numbers?

**Comment by yangwao** [comment]
thinking we should adjust scarcity numbers on very last minute 

image

**Comment by yangwao** [comment]
I think text on item needs to be changed

image

**Comment by yangwao** [comment]
> @yangwao [is very special, kajmak with ketchup](https://github.com/kodadot/nft-gallery/pull/6342/commits/88ab0d35f1c7785f8b87290ac2c6658c20c4982f)

would be boring if we are all same right
image

**Review by yangwao — APPROVED** [review]

**Comment by yangwao** [comment]
Mint success I would put at least for 20 seconds for slow people first time experiencing our site.

**Comment by yangwao** [comment]
image

when is mint success, maybe do redirect? 


well I would say it works now

image

**Pull request merged**